openvdb_houdini/openvdb_houdini/SOP_OpenVDB_Activate.cc

```diff
@@ -490,9 +504,9 @@ sopFillSDF(GridType &grid, int dummy)
 
 template <typename GridType>
 static void
-sopDilateVoxels(GridType& grid, exint count)
+sopDilateVoxels(GridType& grid, exint count, openvdb::tools::NearestNeighbors nn)
 {
-    openvdb::tools::dilateActiveValues(grid.tree(), static_cast<int>(count));
+    openvdb::tools::dilateActiveValues(grid.tree(), static_cast<int>(count), nn);
```

Comment:
worth passing in for the `sopErodeVoxels` method as well?

Reply:
Yes, I think that makes sense.